src/main/java/com/commercetools/sync/customobjects/utils/CustomObjectSyncUtils.java

```diff
@@ -0,0 +1,54 @@
+package com.commercetools.sync.customobjects.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.customobjects.CustomObject;
+import io.sphere.sdk.customobjects.CustomObjectDraft;
+
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CustomObjectSyncUtils {
+
+    /**
+     * Compares the value of a {@link CustomObject} to the value of a {@link CustomObjectDraft}.
+     * It returns a boolean whether the values are identical or not.
+     *
+     * @param oldCustomObject the {@link CustomObject} which should be synced.
+     * @param newCustomObject the {@link CustomObjectDraft} with the new data.
+     * @return A boolean whether the value of the CustomObject and CustomObjectDraft is identical or not.
+     */
+
+    @Nonnull
+    public static boolean hasIdenticalValue(
+            @Nonnull final CustomObject<JsonNode> oldCustomObject,
+            @Nonnull final CustomObjectDraft<JsonNode> newCustomObject) {
+        JsonNode oldValue = oldCustomObject.getValue();
+        JsonNode newValue = newCustomObject.getValue();
+
+        return oldValue.equals(newValue);
+    }
+
+    /**
+     * Given a list of {@link CustomObjectDraft} and a {@code batchSize}, this method distributes the custom
+     * object drafts into batches with the {@code batchSize}. Each batch is represented by a {@link List} of drafts and
+     * all the batches are grouped and represented by a {@link List}&lt;{@link List}&gt; of elements, which is returned
+     * by the method.
+     *
+     * @param elements  the list of custom object drafts to split into batches.
+     * @param batchSize the size of each batch.
+     * @return a list of lists where each list represents a batch of {@link CustomObjectDraft}.
+     */
+
+    @Nonnull
+    public static List<List<CustomObjectDraft<JsonNode>>> batchElements(
```

Comment:
This method is the same as the one in SyncUtils. Why did you create this one?

Reply:
Also, in the tests you're testing `SyncUtils#batchElements()`, not this method. 😉  Maybe call `SyncUtils#batchElements()` from this method?

Reply:
> This method is the same as the one in SyncUtils. Why did you create this one?

SyncUtils accept type List<T> but in CO we need to pass List<CustomObject<JsonNode>>, so I implement it in CustomObjectSyncUtils

Reply:
It is called from `CustomObjectSync`, which is still in progress.
The test case is inside `CustomObjectSyncUtilsTest`

Reply:
What about this?

```
@Nonnull
    public static List<List<CustomObjectDraft<JsonNode>>> batchElements(
            @Nonnull final List<CustomObjectDraft<JsonNode>> elements,
            final int batchSize) {
        return SyncUtils.batchElements(elements, batchSize);
    }
```

And as Ahmet wrote, you don't test this method in `CustomObjectSyncUtilsTest`. Look at the import, you have there `import static com.commercetools.sync.commons.utils.SyncUtils.batchElements;`.

Reply:
Lam is right.
I checked once again. Sync class can call `SyncUtils#batchElements`. Generic type doesn't matter here.
I removed `batchElements` from `CustomObjectSyncUtils`.
https://github.com/commercetools/commercetools-sync-java/pull/561/commits/1a10c90390f95bb2a5658c3303554c3ccdf12e22